WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -1362,75 +1397,89 @@ static GSourceFuncs _handlerIntervention =
             return 0;
         }
 
+        std::string urlValue() const
+        {
+            std::unique_lock<std::mutex> lock{urlData_.mutex};
+            return urlData_.url;
+        }
+
+        std::string urlValue(std::string url)
+        {
+            std::unique_lock<std::mutex> lock{urlData_.mutex};
+            std::swap(urlData_.url, url);
+            return url;
```

Comment:
Returned value is never used, but it's still surprising that setter returns the old member's value.
Consider returning urlData_.url or void.

Reply:
its setter function.